code/upaya/app/assets/javascripts/homepage.js

```diff
@@ -0,0 +1,13 @@
+$(document).on('page:change', function () {
+  var maxH = 0;
+  $('.same-height-panel').each( function () {
+    var thisH = $(this).height();
+    if (thisH > maxH) {
+      maxH = thisH;
+    }
+  });
+
+  $('.same-height-panel').each( function () {
+    $(this).css('minHeight',maxH+'px');
+  });
+});
```

Comment:
This file is specific to the ICAM UI. Unless we're planning on having the same look with the 2 panels side by side, we don't need this file.